torchsynth/globals.py

```diff
@@ -30,12 +34,30 @@ def __init__(
         self.sample_rate = sample_rate
         self.buffer_size = buffer_size
 
+        # Default control rate to 100 times less than audio rate
+        if control_rate is None:
+            self.control_rate = (self.sample_rate // 100).clone().detach()
```

Comment:
1) Let's not use magic numbers, make this something in defaults?
2) If sample_rate % control == 0 maybe we should throw an assert? Because then julian resampling will be expensive?

Reply:
Not using Julian resampling so I don't think it will matter too much if it isn't divisible -- but, I think it's not a bad idea anyway to have it as a default and have an assert. Makes more sense for it to be divisible anyway. Agree on magic numbers

Reply:
What are you using if not Julian?

Reply:
PyTorch Upsample with linear interpolation